pkg/cfg/sanitizer_test.go

```diff
@@ -0,0 +1,46 @@
+package cfg_test
+
+import (
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/stretchr/objx"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func TestTimeSanitizer(t *testing.T) {
+	tm := time.Date(2019, time.November, 26, 0, 0, 0, 0, time.UTC)
+	san, err := cfg.TimeSanitizer(tm)
+
+	assert.NoError(t, err)
+	assert.Equal(t, "2019-11-26T00:00:00Z", san)
+
+	i := 1337
+	san, err = cfg.TimeSanitizer(i)
+
+	assert.NoError(t, err)
+	assert.Equal(t, 1337, san)
+}
+
+func TestSanitize(t *testing.T) {
+	in := map[string]interface{}{
+		"foo":  "bar",
+		"date": time.Date(2019, time.November, 26, 0, 0, 0, 0, time.UTC),
+		"nested": map[string]interface{}{
+			"anotherDate": time.Date(2019, time.November, 26, 0, 0, 0, 0, time.UTC),
+		},
+	}
+
+	san, err := cfg.Sanitize(in, []cfg.Sanitizer{
+		cfg.TimeSanitizer,
+	})
+
+	assert.NoError(t, err)
+	assert.IsType(t, map[string]interface{}{}, san)
+
+	s := objx.New(san)
+
+	assert.Equal(t, "bar", s.Get("foo").Data())
```

Comment:
You should also try to cover the error cases in your code - are they actually reachable / viable?
You don't want to introduce errors while handling errors ;-)